cobbler/cobbler_collections/manager.py

```diff
@@ -12,6 +12,7 @@
 from cobbler.cexceptions import CX
 from cobbler import serializer
 from cobbler import validate
+from cobbler.settings import Settings as Settings
```

Comment:
```suggestion
from cobbler.settings import Settings
```

Reply:
fixed in https://github.com/cobbler/cobbler/pull/3266/commits/d8117a8cb4f51494abf7ebc60d0d9421704cf042